extensions/intellij/intellij_generator_plugin/src/main/resources/templates/rx_bloc_tests/bloc_alchemist_golden.dart.template

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '../factory/${bloc_snake_case}_factory.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rx_bloc_list/models.dart';
+
+
+void main() {
+  runGoldenTests([
+    buildScenario(
+      scenario:  '${bloc_snake_case}_empty',
+      widget: ${bloc_field_case}Factory(),
+    ),
+    buildScenario(
+      scenario:  '${bloc_snake_case}_success',
+      widget: ${bloc_field_case}Factory(),
+    ),
+    buildScenario(
+      scenario:  '${bloc_snake_case}_loading',
+      widget: ${bloc_field_case}Factory(),
```

Comment:
Please utilize the loading argument in the scenario level. @DDavidPrime or @GoranPrime can assist here if needed

Reply:
Done